src/client/components/QuickPostEditor/QuickPostEditorFooter.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+import classNames from 'classnames';
+import { Icon } from 'antd';
+import Action from '../Button/Action';
+
+const QuickPostEditorFooter = ({
+  currentImages,
+  imageUploading,
+  postCreationLoading,
+  handleCreatePost,
+  handleImageChange,
+  postText,
+  submittingPostText,
+  onRemoveImage,
+  handleFooterFocus,
+}) => (
+  // eslint-disable-next-line
```

Comment:
Can we change this to reference one (or more) rule instead of disabling eslint entirely?

Reply:
oh yeah will update comment